flexget/plugins/urlrewrite_nnmclub.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+from flexget.utils import requests
```

Comment:
One last thing lol. We generally have the imports in 3 categories. "Standard python libs", "other libs" and "flexget imports" :P